toolkits/global/packages/global-card/scss/50-components/card.scss

```diff
@@ -94,6 +87,11 @@
 			margin-bottom: 0;
 		}
 	}
+
+	a {
+		// WCAG 1.4.1 Use of Color
+		text-decoration: underline;
```

Comment:
this feels like something that should be in the CSS further up the stylesheet. I'm assuming it's nuked somewhere?

Is `text-decoration: none;` something we should remove wherever it is near the top of the Sass imports?

I'd have expected that `text-decoration: underline;` be the norm, not the exception.

Reply:
It is indeed removed at a high level (https://github.com/springernature/frontend-toolkits/blob/features/design-tokens/context/brand-context/springer/scss/40-base/basic.scss#L11 for springer)

Reply:
I think we can resolve this for now, to get the merged. 

But I think this sort of a11y thing should be discussed further and implemented accordingly. 

If it looks like a link, and links to a page like a link, it's a link. If it's a node of text that looks the same as every other bit of text you won't know if it's a link unless you hover it (I guess) and we'd want our site visitors to be reading the content, not rushing their mouse cursor over it to find any potential links :sob: